src/httprpc.cpp

```diff
@@ -119,14 +122,17 @@ static bool multiUserAuthorized(std::string strUserPass)
             std::string strHashFromPass = HexStr(hexvec);
 
             if (TimingResistantEqual(strHashFromPass, strHash)) {
+                if (vFields.size() > 3) {
+                    walletNameOut = vFields[3];
```

Comment:
In commit "RPC: Allow rpcauth configs to specify..."

I think instead of interpreting the 4th rpcauth field as a wallet filename field, it might be better to treat it as a generic options field (similar to the field in `fstab` files for mount options). E.g. instead of:

```
rpcauth=user:salt:hash:filename.dat
```

You would write:

```
rpcauth=user:salt:hash:wallet=filename.dat
```

This would be more extensible, also more readable.